core/assets/bundles/bundle.properties

```diff
@@ -623,7 +623,7 @@ mech.javelin-ship.description = A hit-and-run strike ship. While initially slow,
 mech.javelin-ship.weapon = Burst Missiles
 mech.javelin-ship.ability = Discharge Booster
 mech.trident-ship.name = Trident
-mech.trident-ship.description = A heavy bomber. Reasonably well armored.
+mech.trident-ship.description = A heavy bomber, majorly effective against buildings and ground based targets; although it does not have any means of protection from air-based enemies. Reasonably well-armoured.
```

Comment:
I beg to differ. Have you tried bombing air units? It absolutely ANNIHILATES them hahaha